CheckTheFridge/ClientApp/src/components/AddIngredient.js

```diff
@@ -20,15 +21,39 @@ export function AddIngredient() {
         }
     }, [])
 
+    async function addIngredient2(ingredient) {
+
+        //console.log("Ingredient.form data: ", ingredient.name, ingredient.description)
+        //const id = uuidv4();
+        //ingredient.id = id;
+
+        await fetch('Ingredient/Add/' + ingredient.name + '/' + ingredient.description + '/' + ingredient.id + '/' + ingredient.quantity, { method: 'POST' })
+            .then((response) => {
+                if (response.ok) {
+                    console.log('Ingredient created')
+                }
+                else
+                    throw new Error('Ingredient not created');
```

Comment:
This is what I am getting back as an error when I try to add an ingredient. Good job at error handling!